app/src/main/java/io/homeassistant/companion/android/settings/SettingsActivity.kt

```diff
@@ -49,6 +69,16 @@ class SettingsActivity : BaseActivity() {
         setSupportActionBar(findViewById(R.id.toolbar))
         supportActionBar?.setDisplayHomeAsUpEnabled(true)
 
+        blurView = findViewById(R.id.blurView)
+        blurView.setupWith(getWindow().getDecorView().getRootView() as ViewGroup)
+            .setBlurAlgorithm(RenderScriptBlur(this))
+            .setBlurAutoUpdate(true)
+            .setBlurRadius(8f)
+            .setHasFixedTransformationMatrix(false)
+            .setBlurEnabled(false)
```

Comment:
I'm seeing differences with the `WebViewActivity` that seem unnecessary:

 - Why make it complicated with the window decorview root instead of using `findViewById(R.id.root)`?
 - Auto updating is enabled by default according to the documentation

Reply:
When adding the blurview to the settingsActivity, the fragment content kept showing up on top of the blur. I've made the auto update explicit to ensure the blur is redrawn on content changes, though as you indicate it seems to be redundant.
The rootView with the added background finally caused the blur to show as intended.

Reply:
Please see [this comment](https://github.com/home-assistant/android/pull/2926#discussion_r985075537) - with the latest code, the background doesn't appear to be necessary for the blur to work and has other unwanted side effects.

For code consistency, switching to property access (first point) is still preferred.